### ISSUE TITLE
Add a license file to github

### ISSUE THREAD

**Issue by flatsiedatsie** [mysensors/MySensors#1122]
With a license file it becomes easier to find and sort for projects with a certain open source license.

**Comment by mandarcthorat** [comment]
As MySensors is an open source hardware and software community, it would be great if this code is released under permissive license(Ex. MIT or BSD3).

**Comment by mfalkvidd** [comment]
MySensors current license text: 
https://www.mysensors.org/about/mysensors#copyright

**Comment by flatsiedatsie** [comment]
@mfalkvidd Thanks, but this issue is about creating a license file in the Github code itself.

**Comment by mfalkvidd** [comment]
@flatsiedatsie yes I am aware of that. But it seemed from mandarcthorat's comment that _which_ license to use was up for discussion.

**Comment by mandarcthorat** [comment]
@mfalkvidd  Thanks for sharing the relevant link.
